### PR TITLE
[Merged by Bors] - feat: indicator of a clopen set as a bounded continuous function

### PR DESCRIPTION
**Pull request by YaelDillies**
From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Counterexamples/Phillips.lean [hunk 1]
```diff
@@ -385,7 +385,7 @@ section
 
 
 theorem norm_indicator_le_one (s : Set α) (x : α) : ‖(indicator s (1 : α → ℝ)) x‖ ≤ 1 := by
-  simp only [indicator, Pi.one_apply]; split_ifs <;> norm_num
+  simp only [Set.indicator, Pi.one_apply]; split_ifs <;> norm_num
```
Comment (Ruben-VandeVelde) [inline Counterexamples/Phillips.lean:388]: Is this a (weak) indication that the new one should be protected?

**Review by Ruben-VandeVelde — APPROVED** [review]
Simple enough

maintainer merge

File: Counterexamples/Phillips.lean (hunk 1 above)
Comment (YaelDillies) [reply Counterexamples/Phillips.lean:388]: Maybe, but in general I think it's fine to have both `A.X` and `B.X` unprotected.

**Review by j-loreaux — COMMENTED** [review]
Thanks!

bors merge

**Pull request closed without merging**